incubator/elk/requirements.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: kibana
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  version: 0.2.0
+- name: logstash
+  repository: https://kubernetes-charts-incubator.storage.googleapis.com/
+  version: 0.3.0
```

Comment:
Version does not match the `requirements.yaml` version. Please updated the dependencies with the helm command `helm dependency update`.

Reply:
Damn. I will fix.